airbyte-integrations/connectors/source-mysql-strict-encrypt/src/test-integration/java/io/airbyte/integrations/source/mysql_strict_encrypt/AbstractMySqlSslCertificateStrictEncryptSourceAcceptanceTest.java

```diff
@@ -37,8 +39,8 @@ protected void setupEnvironment(final TestDestinationEnv environment) throws Exc
         .put("method", "STANDARD")
         .build());
     config = Jsons.jsonNode(ImmutableMap.builder()
-        .put(JdbcUtils.HOST_KEY, innerContainerAddress.left)
-        .put(JdbcUtils.PORT_KEY, innerContainerAddress.right)
+        .put(JdbcUtils.HOST_KEY, HostPortResolver.resolveHost(container))
+        .put(JdbcUtils.PORT_KEY, HostPortResolver.resolvePort(container))
```

Comment:
Does this still work when running directly on macOS with gradle `:integrationTestJava`?